raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -815,16 +815,26 @@ def test_matrix_invitee_receives_invite_on_restart(matrix_transports):
 
 @pytest.mark.parametrize("matrix_server_count", [3])
 @pytest.mark.parametrize("number_of_transports", [3])
-def test_matrix_user_roaming(matrix_transports):
+@pytest.mark.parametrize(
+    "roaming_peer",
+    [pytest.param("high", id="roaming_high"), pytest.param("low", id="roaming_low")],
+)
+def test_matrix_user_roaming(matrix_transports, roaming_peer):
     transport0, transport1, transport2 = matrix_transports
     received_messages0 = set()
     received_messages1 = set()
 
     message_handler0 = MessageHandler(received_messages0)
     message_handler1 = MessageHandler(received_messages1)
 
-    raiden_service0 = MockRaidenService(message_handler0)
-    raiden_service1 = MockRaidenService(message_handler1)
+    privkeys = sorted([make_privatekey_bin(), make_privatekey_bin()])
```

Comment:
How does this make sure they're ordered?

Reply:
Another good point. It orders the private keys but that of course doesn't guarantee a specific ordering of the corresponding addresses.